__tests__/app.test.js

```diff
@@ -0,0 +1,38 @@
+const request = require('supertest')
+const app = require('../app')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const { articleData, commentData, topicData, userData} = require('../db/data/test-data/index')
+
+beforeEach(() => {
+    return seed({ topicData, userData, articleData, commentData })
+})
+
+afterAll(() => {
+    db.end()
+})
+
+describe('/api/topics', () => {
+    describe('GET', () => {
+        test('GET 200: responds with an array of all topics in the correct format', () => {
```

Comment:
This test description alludes to 2 behaviours being tested - all topics being returned *and* topics returned are in correct format. As these are different behaviours, it would be good to split these into 2 different tests